src/CodeCoverage/Generators/HtmlGenerator.php

```diff
@@ -95,8 +95,13 @@ private function parse(): void
 			}
 
 			$light = $total ? $total < 5 : count(file($entry)) < 50;
+			$nameReplacePairs = [];
```

Comment:
There should be single replacement, common path of all sources. Otherway duplicates may appear.
And it can be done before `foreach` loop.

Reply:
@milo What is the easiest way to find common path of all sources?

Reply:
[This code](https://github.com/nette/tester/blob/ca80e7bb1210c5e2141925b8f74ba7cf43500843/src/CodeCoverage/Generators/AbstractGenerator.php#L55-L63) can be reused.